shared/dataflow/codeql/dataflow/TaintTracking.qll

```diff
@@ -0,0 +1,83 @@
+/**
+ * Provides classes for performing local (intra-procedural) and
+ * global (inter-procedural) taint-tracking analyses.
+ */
+
+import DataFlow
+import DataFlowImpl
+import DataFlowParameter
+import TaintTrackingParameter
+
+module TaintFlowMake<
+  DataFlowParameter DataFlowLang, TaintTrackingParameter<DataFlowLang> TaintTrackingLang>
+{
+  private import DataFlowLang
```

Comment:
Is this line necessary?

Reply:
I think this is a left-over from an earlier iteration in which I hadn't realised I needed the imports below. Will remove.

Reply:
Removed.